From 07004baecebf9c0d5d2b4ee48f38a15439138a54 Mon Sep 17 00:00:00 2001
From: Nanda Kishore Chinnaram <nanda_kishore_chinna@dell.com>
Date: Thu, 5 Nov 2015 20:00:44 +0530
Subject: [PATCH] Add PCI IDs of Intel Sunrise Point-H SATA Controller S232/236
Patch-mainline: Not yet, to be submitted by Intel
References: bsc#953796

PCI IDs of Intel Sunrise Point chipset(C232/C236) based SATA Controller were missing in AHCI driver.
Due to this, the virtual disks created using the controller were not enumerated during linux installation.

Signed-off-by: Nanda Kishore Chinnaram <nanda_kishore_chinna@dell.com>
Acked-by: Michal Marek <mmarek@suse.com>
---
 drivers/ata/ahci.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/ata/ahci.c b/drivers/ata/ahci.c
index ff02bb4..d56847f 100644
--- a/drivers/ata/ahci.c
+++ b/drivers/ata/ahci.c
@@ -354,6 +354,8 @@ static const struct pci_device_id ahci_pci_tbl[] = {
 	{ PCI_VDEVICE(INTEL, 0xa105), board_ahci }, /* Sunrise Point-H RAID */
 	{ PCI_VDEVICE(INTEL, 0xa107), board_ahci }, /* Sunrise Point-H RAID */
 	{ PCI_VDEVICE(INTEL, 0xa10f), board_ahci }, /* Sunrise Point-H RAID */
+	{ PCI_VDEVICE(INTEL, 0xa106), board_ahci }, /* Sunrise Point-H RAID */
+	{ PCI_VDEVICE(INTEL, 0xa102), board_ahci }, /* Sunrise Point-H AHCI */
 
 	/* JMicron 360/1/3/5/6, match class to avoid IDE function */
 	{ PCI_VENDOR_ID_JMICRON, PCI_ANY_ID, PCI_ANY_ID, PCI_ANY_ID,
-- 
2.4.3

